src/lib/dataEntities/networks.ts

```diff
@@ -28,10 +28,14 @@ export interface L1Network extends Network {
 }
```

Comment:
It will be left as `false` here because `L1Network` doesn't include Arbitrum chains. If an Arbitrum chain is a parent chain, it will be part of the `ParentChain`.

Reply:
https://github.com/OffchainLabs/arbitrum-sdk/pull/319 added a util method to detect if a chain is Arbitrum chain runtime